CHANGES.md

```diff
@@ -137,6 +137,8 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 
 * DC/OS UI X-Frame-Options default value has been changed from `SAMEORIGIN` to `DENY`. This is now configurable using the `adminrouter_x_frame_options` configuration value (DCOS-49594)
 
+* HTTP endpoint targets of DC/OS Diagnostics can be marked optional (DCOS_OSS-5031)
```

Comment:
Is there documentation for this anywhere?

Reply:
No, I didn't see any existing documentation set to update for dcos-diagnostics. If you can link me to an existing doc I'll be happy to update it.

Reply:
@janisz is there anything like it?

Reply:
Nope, it's internal config. But yeah we should document it in dcos-diagnostics repo